.cspell.json

```diff
@@ -40,6 +40,16 @@
     "\\[.*/.*\\]\\(https://github.com",
     "github.com[/:][\\w._\\-]+(/[\\w._\\-]+)?"
   ],
+  "languageSettings": [
+    {
+      "languageId": "markdown",
+      "dictionaries": [
+        "cpp",
+        "latex",
```

Comment:
Why did you add latex? I think it's not used in autoware.

Reply:
It's used in Planning/Control documents.
https://github.com/autowarefoundation/autoware.universe/blob/4536e52bdbc80ea3dec0d2940ded51ffd118b477/planning/obstacle_avoidance_planner/obstacle_avoidance_planner-design.ja.md

Reply:
I see. Thanks!